client/src/features/IteamX/Hero.js

```diff
@@ -1,52 +1,16 @@
 import React from 'react'
-import { Container, Row, Col } from 'react-bootstrap'
 
-import { Section, Box, Typography, Span, CTALink } from '../../components/Core'
-import { getExternalOrInternalLink } from '../../utils/helpers'
+import { Typography, Span } from '../../components/Core'
+import { BaseHero } from '../Hero'
 
-export const Hero = ({ content }) => {
-  const [title] = content.title.split('.')
+export const Hero = ({ title, subtitle, mediaType, link }) => {
+  const splitTitle = title.split('.')
```

Comment:
What does this do?
looks like `splitTitle` is now an array.

Reply:
Yes, strange how it worked anyway. I'll add the array destructuring again.